### ISSUE TITLE
Crashes in Mat.clone()

### ISSUE THREAD

**Issue by jpsacha** [bytedeco/javacpp-presets#1014]
I am getting JVM crashes in v.1.5.4 on calls to `Mat.clone()` with `EXCEPTION_ACCESS_VIOLATION` errors
```
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ff90359122e, pid=15020, tid=4024
```

The crashes are coming from the same static method in [ijp-javacv toMat()](https://github.com/ij-plugins/ijp-javacv/blob/0420647c7bd673b7b1d813bdcdc8d52a65299dbb/ijp-javacv-core/src/main/scala/net/sf/ij_plugins/javacv/IJOpenCVConverters.scala#L169):
```scala
  def toMat(src: ImagePlus): Mat = {
    val frame = new ImagePlusFrameConverter().convert(src)
    val converter = new OpenCVFrameConverter.ToMat()
    converter.convert(frame).clone()
  }
```
I use 'clone()' following discussion of issue "Crash on cvtColor" #979. `converter` is still in scope here when the crash happens.

Error locations from the dump:
```
Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 3839  org.bytedeco.opencv.opencv_core.Mat.clone()Lorg/bytedeco/opencv/opencv_core/Mat; (0 bytes) @ 0x000001b2f38711fb [0x000001b2f38711c0+0x000000000000003b]
J 3945 c1 net.sf.ij_plugins.javacv.IJOpenCVConverters$.toMat(Lij/ImagePlus;)Lorg/bytedeco/opencv/opencv_core/Mat; (44 bytes) @ 0x000001b2ec6ee194 [0x000001b2ec6edce0+0x00000000000004b4]
```
The error type `EXCEPTION_ACCESS_VIOLATION` and location is very consistent. It may take a couple hundred iterations for error to happen (not very consistent here).

The caller code is executed in parallel. Calls to `toMat` may be happening  a couple times a second (5-6). I am am allocating 24G memory (-Xmx24g), process is utilizing only about 8G. Images are up to 4,000x5,000.

Looking for some ways to prevent this happening.

**Comment by saudet** [comment]
Please try to call the new close() method I've added for that in commit
https://github.com/bytedeco/javacv/commit/172cf9916afe4f159fe18dca6c9e27d5b600b127

**Comment by saudet** [comment]
Yes, that's what is happening here. The reference to `frame` is lost after calling `converter.convert(frame)`, and may get garbage collected, which would cause the subsequent call to `Mat.clone()` to crash. Doing something like this will fix that:
```scala
  def toMat(src: ImagePlus): Mat = {
    val frame = new ImagePlusFrameConverter().convert(src)
    val converter = new OpenCVFrameConverter.ToMat()
    val mat = converter.convert(frame).clone()
    converter.close()
    frame.close()
  }
```

**Comment by saudet** [comment]
And this has been released with JavaCV 1.5.5, so you don't need to rely on snapshots!
